src/popup/router/components/AccountInfo.vue

```diff
@@ -3,25 +3,25 @@
     <div class="title">
```

Comment:
I suggest to create one computed prop (eg.: `currentAccount`) to replace all `accounts[i]`.